mantidproject/mantid
# Release Notes for Harmonized Plotting GUI

Signed-off-by: Karl Palmen <karl.palmen@stfc.ac.uk>

Description of work.

The release notes for issue #18159 with pull request #19237 .

**To test:**

Build documentation and check release notes for UI & Usability Changes

Fixes #19259 

Best merged when #18159 is merged.

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
